dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/DependencyConflict.java

```diff
@@ -50,13 +54,15 @@ public DependencyPath getPathToSelectedArtifact() {
   public String toString() {
     Artifact selected = pathToSelectedArtifact.getLeaf();
     Artifact unselected = pathToUnselectedArtifact.getLeaf();
-
+    //    com.google:foo:1 (selected in the class path) does not have symbol 'java.lang.Object's
+    // method equals(Object arg1)' but version '2' has it.
```

Comment:
no need to use quote marks here.

Reply:
This version part is replaced with artifact coordinates without quotes, because now the two artifact coordinates are placed a bit far.